SPECS/kernel/kernel.spec

```diff
@@ -391,6 +391,12 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_sysconfdir}/bash_completion.d/bpftool
 
 %changelog
+* Wed Aug 17 2022 Cameron Baird <cameronbaird@microsoft.com> - 5.15.60.2-1
+- Upgrade to 5.15.60.2 to fix arm64 builds
+
+* Wed Aug 17 2022 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 5.15.60.1-1
```

Comment:
Nit: Not sure you need the 5.15.60.1 in the changelog since it never technically went in

Reply:
@Camelron please remove the 5.15.60.1 entries to keep our changelogs clean